.travis.yml

```diff
@@ -1,3 +1,5 @@
+sudo: required
```

Comment:
I don't see any code here that requires such change.

Based on this information http://docs.travis-ci.com/user/workers/standard-infrastructure/ I don't see any reason to use this. Please explain